build.gradle

```diff
@@ -46,21 +46,21 @@ jar {
 }
 
 dependencies {
+    api platform('com.exactpro.th2:bom:4.0.2')
+    implementation 'com.exactpro.th2:common:5.0.0-dev-version-5-3838510969-SNAPSHOT'
+    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
 
-    implementation 'com.exactpro.th2:common:3.31.1'
-
-    implementation "org.slf4j:slf4j-log4j12"
-    implementation "org.slf4j:slf4j-api"
+    implementation 'org.slf4j:slf4j-log4j12'
+    implementation 'org.slf4j:slf4j-api'
 
-    implementation group: 'org.apache.kafka', name: 'kafka_2.13', version: '2.8.0'
-    implementation 'io.reactivex.rxjava3:rxjava:3.0.4'
+    implementation 'org.apache.kafka:kafka_2.13:3.3.1'
+    implementation 'io.github.microutils:kotlin-logging:3.0.4'
```

Comment:
This version uses kotlin 1.7. Please, use older version that depend on kotlin 1.6